### PR TITLE
Initialize components with React

### PR DESCRIPTION
**Pull request by AlexisJW**
In this Bookstore project, The task is to initialize the project with components.

This pull request concerns the list of tasks according to the [assignment](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v3_0.md) and the steps following:

Project requirements

- Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- Create a directory for your reusable /components.
- Add [React Router](https://reactrouter.com/en/main/start/tutorial#setup)
- [Setup your router](https://reactrouter.com/en/main/start/tutorial#adding-a-router)
- Add two Routes to your application:
-   /books - the default view:
      Should display a list of books
      Should display a form for adding a book (no functionality yet).
-   /categories:
      Should display a button "Check status" only.
- Create a <Navbar> component that includes <Link>s to the two routes you created
- Create a <Book> component that displays a single book (props are title and author.
  Each Book has a "remove" button (no functionality yet)
- Create a <Form> component (with inputs for a title and an author [IMPORTANT in the design you can see an input for a category).

** Note: Styling is NOT required at this state

### CODE REVIEW

**Review by fobadara — COMMENTED** [review]
Hi @AlexisJW, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the future features section and it has a live demo section without a live demo link

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by AlexisJW** [comment]
Hi @fobadara,

Thank you for taking the time to review my project and for your feedback. I appreciate your effort and your valuable insights.

I understand that the project did not meet the expected criteria, as it lacks the future features section and a live demo link, I'm don't understand that this has made the code review request invalid.

Please, pay attention again to the requirement.

Sincerely, 👍

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @AlexisJW 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑🛑
yPkfDFf

- [ ] Please consider removing non-important contents from your readme, The template is designed by Microverse for us all to use to document our projects professionally and has it filled with content related to each project we may be working on. currently, I can see you have not removed the CREATE REACT APP Reamde content, And some section are missing. 
> Please update yoour readme to include all section and remove non related contents. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by AlexisJW** [comment]
Hi @TadesseDev,

Thank you for your time and constructive feedback. I'll do it.

File: README.md [hunk 1]
```diff
@@ -1 +1,232 @@
...
+
+  - Your welcome, copy this project and let me know if you need something.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```
Comment (rotimiazeez) [inline README.md:230]: - Please note that the MIT license link provided is broken, kindly fix it 👍🏽

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @AlexisJW  👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Review by EricMbouwe — APPROVED** [review]
Hi @AlexisJW, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Requested changes ✔
- No linter errors 
## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**